ansible_collections/arista/avd/plugins/modules/eos_designs_facts.py

```diff
@@ -30,13 +30,47 @@
     so all devices can lookup values of any other device without using the slower `hostvars`.
   - The facts can also be copied to the "root" `switch` in a task run per-device (see example below)
   - The module is used in `arista.avd.eos_designs` to set facts for devices, which are then used by jinja templates
-    in `arista.avd.eos_designs` to generate the `structured_configuration`.
+    and python module in `arista.avd.eos_designs` to generate the `structured_configuration`.
 options:
-  avd_switch_facts:
```

Comment:
so we are removing the possibility to not generate the switch_facts? if so it should probably need to go in some changelog somewhere

Reply:
this was never meant to be a real option. The plugin has no function. It was only added as a knob to satisfy @carlbuchmann 's OCD of not having any options :)

Reply:
ack - hopefully nobody is using this out there :) resolving